tests/platform/conftest.py

```diff
@@ -1,7 +1,82 @@
 import pytest
+from common.fixtures.advanced_reboot import get_advanced_reboot
 
 @pytest.fixture(autouse=True, scope="module")
 def skip_on_simx(testbed_devices):
     platform = testbed_devices["dut"].facts["platform"]
     if "simx" in platform:
-        pytest.skip('skipped on this platform: {}'.format(platform))
\ No newline at end of file
+        pytest.skip('skipped on this platform: {}'.format(platform))
+
+# Platform pytest arguments
+def pytest_addoption(parser):
```

Comment:
(Probably not in scope for this PR) but between the sub-folder `conftest` files and the main `conftest` file I feel like we're going to end up with a loooooot of command line options in the not-so-distant future. Is there a better way to capture these test-case specific options? @yxieca @wangxin thoughts?

Reply:
moved them to separate args dir and included it in conftest

Reply:
move them to new args dir under platform. this should limit the number of flat args by breaking them into submodules.

Reply:
I like that solution. :)